content/en/users/aai/check-in/faq/_index.md

```diff
@@ -9,6 +9,19 @@ description: >
   Most frequent questions about EGI Check-in
 ---
 
+## Getting help
+
+Requests for technical assistance and other issues related to Check-in can be
+obtained by sending an e-mail to:
+
+**checkin-support@mailman.egi.eu**
+
+Alternatively, if you already have a Check-in account, the [EGI HelpDesk](https://ggus.eu/)
+can be used. In that case, submit a new ticket and make
+sure that the support unit is assigned to "Check-in (AAI)".
```

Comment:
```suggestion
sure that the support unit is assigned to **Check-in (AAI)**.
```
Bold text may be more visual, as you feel.

Reply:
(When adding the documentation link I also fixed the Helpdesk capitalisation)